.travis.yml

```diff
@@ -47,4 +47,4 @@ script:
   - moodle-plugin-ci grunt
   - moodle-plugin-ci phpdoc
   - moodle-plugin-ci phpunit
-  - moodle-plugin-ci behat
\ No newline at end of file
+  - moodle-plugin-ci behat
```

Comment:
can you please remove this carriage return change? - Ideally a patch should only touch files that are associated with the change. thanks!